pkg/exchange/binance/binanceapi/place_margin_order_request.go

```diff
@@ -0,0 +1,26 @@
+package binanceapi
+
+import (
+	"github.com/c9s/requestgen"
+)
+
+//go:generate requestgen -method POST -url "/sapi/v1/margin/borrow-repay" -type PlaceMarginOrderRequest -responseType .TransferResponse
+type PlaceMarginOrderRequest struct {
+	client requestgen.AuthenticatedAPIClient
+
+	asset string `param:"asset"`
+
+	// TRUE for Isolated Margin, FALSE for Cross Margin, Default FALSE
+	isIsolated bool `param:"isIsolated"`
+
+	// Only for Isolated margin
+	symbol *string `param:"symbol"`
+
+	amount string `param:"amount"`
```

Comment:
use fixedpoint value?

Reply:
OK.

The previous type is because I simply copied the value from another file.
The caller uses `req.Price(order.Market.FormatPrice(order.Price))`